<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>857</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <widget class="QWidget" name="stackedWidgetPage1">
       <widget class="QLabel" name="welcomeLabel">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>180</y>
          <width>175</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>欢迎使用智能校园健康管理系统</string>
        </property>
       </widget>
       <widget class="QPushButton" name="generateButton">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>350</y>
          <width>105</width>
          <height>32</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>开始生成计划</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="idInput">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>290</y>
          <width>125</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="nameInput">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>230</y>
          <width>125</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="nameLabel">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>230</y>
          <width>58</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>姓名：</string>
        </property>
       </widget>
       <widget class="QLabel" name="idLabel">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>290</y>
          <width>58</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>学号：</string>
        </property>
       </widget>
       <widget class="QLabel" name="welcomeLabel_2">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>40</y>
          <width>391</width>
          <height>101</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt; font-weight:700; font-style:italic;&quot;&gt;Weiming Coders&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt; font-weight:700; font-style:italic;&quot;&gt;智能校园健康管理系统&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>-60</y>
          <width>221</width>
          <height>231</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>resource/Unknown.jpeg</pixmap>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="stackedWidgetPage2">
       <widget class="QLineEdit" name="ageInput">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>20</y>
          <width>113</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>年龄</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="heightInput">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>70</y>
          <width>125</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>身高（cm）</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="weightInput">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>130</y>
          <width>125</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>体重（kg）</string>
        </property>
       </widget>
       <widget class="QComboBox" name="goalCombo">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>200</y>
          <width>103</width>
          <height>32</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;健康目标&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="whatsThis">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;健康目标&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="placeholderText">
         <string>健康目标</string>
        </property>
        <property name="duplicatesEnabled">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>减脂</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>增肌</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>健康</string>
         </property>
        </item>
       </widget>
       <widget class="QComboBox" name="dietCombo">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>260</y>
          <width>103</width>
          <height>32</height>
         </rect>
        </property>
        <property name="currentText">
         <string/>
        </property>
        <property name="placeholderText">
         <string>饮食偏好</string>
        </property>
        <item>
         <property name="text">
          <string>荤食</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>素食</string>
         </property>
        </item>
       </widget>
       <widget class="QListWidget" name="freeTimeList">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>330</y>
          <width>211</width>
          <height>141</height>
         </rect>
        </property>
        <property name="whatsThis">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;空闲时间（可多选）&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="accessibleDescription">
         <string/>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SelectionMode::MultiSelection</enum>
        </property>
        <item>
         <property name="text">
          <string>1-2节课</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3-4节课</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5-6节课</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7-8节课</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>9-10节课</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>357</x>
          <y>310</y>
          <width>111</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>空余时间（可多选）</string>
        </property>
       </widget>
       <widget class="QPushButton" name="submitButton">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>490</y>
          <width>141</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>生成个性化健康计划</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="stackedWidgetPage3">
       <widget class="QTextEdit" name="workoutResult">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>110</y>
          <width>201</width>
          <height>131</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextEdit" name="mealResult">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>310</y>
          <width>201</width>
          <height>131</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>280</y>
          <width>111</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:700;&quot;&gt;饮食建议&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>80</y>
          <width>111</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:700;&quot;&gt;运动建议&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
